src/streamlit_extras/tags/__init__.py

```diff
@@ -0,0 +1,207 @@
+        "50": "#ffd16a",
+        "40": "#ffe08e",
+        "30": "#ffecb0",
+        "20": "#fff6d0",
+        "10": "#fffae8",
+    },
+    "yellow": {
+        "100": "#dea816",
+        "90": "#edbb16",
+        "80": "#faca2b",
+        "70": "#ffe312",
+        "60": "#fff835",
+        "50": "#ffff59",
+        "40": "#ffff7d",
+        "30": "#ffffa0",
+        "20": "#ffffc2",
+        "10": "#ffffe1",
+    },
+    "green": {
+        "100": "#177233",
+        "90": "#158237",
+        "80": "#09ab3b",
+        "70": "#21c354",
+        "60": "#3dd56d",
+        "50": "#5ce488",
+        "40": "#7defa1",
+        "30": "#9ef6bb",
+        "20": "#c0fcd3",
+        "10": "#dffde9",
+    },
+    "blue-green": {
+        "100": "#246e69",
+        "90": "#2c867c",
+        "80": "#29b09d",
+        "70": "#00d4b1",
+        "60": "#20e7c5",
+        "50": "#45f4d5",
+        "40": "#6bfde3",
+        "30": "#93ffee",
+        "20": "#bafff7",
+        "10": "#dcfffb",
+    },
+    "light-blue": {
+        "100": "#15799e",
+        "90": "#0d8cb5",
+        "80": "#00a4d4",
+        "70": "#00c0f2",
+        "60": "#24d4ff",
+        "50": "#4be4ff",
+        "40": "#73efff",
+        "30": "#9af8ff",
+        "20": "#bffdff",
+        "10": "#e0feff",
+    },
+    "blue": {
+        "100": "#004280",
+        "90": "#0054a3",
+        "80": "#0068c9",
+        "70": "#1c83e1",
+        "60": "#3d9df3",
+        "50": "#60b4ff",
+        "40": "#83c9ff",
+        "30": "#a6dcff",
+        "20": "#c7ebff",
+        "10": "#e4f5ff",
+    },
+    "violet": {
+        "100": "#3f3163",
+        "90": "#583f84",
+        "80": "#6d3fc0",
+        "70": "#803df5",
+        "60": "#9a5dff",
+        "50": "#b27eff",
+        "40": "#c89dff",
+        "30": "#dbbbff",
+        "20": "#ebd6ff",
+        "10": "#f5ebff",
+    },
+    "gray": {
+        "100": "#0e1117",
+        "90": "#262730",
+        "80": "#555867",
+        "70": "#808495",
+        "60": "#a3a8b8",
+        "50": "#bfc5d3",
+        "40": "#d5dae5",
+        "30": "#e6eaf1",
+        "20": "#f0f2f6",
+        "10": "#fafafa",
+    },
+}
+
+
+HEADER_COLOR_CYCLE = itertools.cycle(
+    [
+        "light-blue-70",
+        "orange-70",
+        "blue-green-70",
+        "blue-70",
+        "violet-70",
+        "red-70",
+        "green-70",
+        "yellow-80",
+    ]
+)
+
+_SUPPORTED_COLORS = Literal[
+    "light-blue-70",
+    "orange-70",
+    "blue-green-70",
+    "blue-70",
+    "violet-70",
+    "red-70",
+    "green-70",
+    "yellow-80",
+]
+
+_DEFAULT_COLOR = "#808495"
+
+
+@extra
+def tagger_component(content: str, tags: list, color_name=None):
+    """
+    Displays tags next to your text.
+    Args:
+        content (str): Content to be tagged
+        tags (list): A list of tags to be displayed next to the content
+        color_name: choose from 'light-blue-70', orange-70, blue-green-70, 'blue-70', 'violet-70', 'red-70', 'green-70' and 'yellow-80'
```

Comment:
What about changing these to just be light-blue, orange, blue-green, etc., so that you don't have to type the 70/80? Since you can't change to, say, orange-80, seems like just 'orange' should be enough.

Reply:
Also, could you make it so that you could give a single color instead of a list? That way you could just do `color="green"` to make them all green?

Reply:
single color support is added!